src/pages/index.js

```diff
@@ -133,3 +250,21 @@ profileEditButton.addEventListener("click", () => {
 addNewCardButton.addEventListener("click", () => {
   newCardPopup.open();
 });
+
+// edit avatar pop up
+editAvatarButton.addEventListener("click", () => {
+  const currentAvatar = document.querySelector(".profile__image").src;
+  document.querySelector("#edit-avatar-link-input").value = currentAvatar;
```

Comment:
You don't need it for the avatar. It's beter to leave the input empty

Or you need to use the method of `UserInfo` here